src/store/index.js

```diff
@@ -61,11 +61,14 @@ const store = createStore({
       try {
         const request = await axiosInstance.post('/me');
 
-        commit('setUser', request.data.user);
+        commit('setUser', request.data);
       } catch (error) {
         commit('removeToken');
       }
     },
+    showUser({ getters }, payload) {
```

Comment:
`getUser` instead of `showUser` is a better choice for this method name. This method doesn't show any user, but it does fetch/get it